.circleci/config.yml

```
@@ -203,35 +215,34 @@ jobs:
             - example/android/app/build/outputs/apk/*
 
   "Test: Android e2e":
-    <<: *macos_defaults
+    <<: *macos_defaults_android
     steps:
       - *addWorkspace
       - run:
           name: Configure env variables
           command: |
             echo 'export ANDROID_HOME="/usr/local/share/android-sdk"' >> $BASH_ENV
             echo 'export ANDROID_SDK_ROOT="/usr/local/share/android-sdk"' >> $BASH_ENV
-            echo 'export PATH="$ANDROID_SDK_ROOT/emulator:$ANDROID_SDK_ROOT/tools:$ANDROID_SDK_ROOT/platform-tools:$ANDROID_SDK_ROOT/tools/bin:$PATH"' >> $BASH_ENV
+            echo 'export PATH="$ANDROID_SDK_ROOT/emulator:$ANDROID_SDK_ROOT/tools:$ANDROID_SDK_ROOT/platform-tools:$PATH"' >> $BASH_ENV
             echo 'export QEMU_AUDIO_DRV=none' >> $BASH_ENV
-            echo 'export JAVA_HOME=/Library/Java/Home' >> $BASH_ENV
+            echo 'export JAVA_HOME=$(/usr/libexec/java_home)' >> $BASH_ENV
             source $BASH_ENV
-
-    # Android tools
       - restore-cache: *cache_restore_brew_android
       - run:
           name: Install Android SDK tools
           command: |
+            brew update --preinstall
             brew bundle --file=.circleci/Brewfile.android --no-lock
       - save-cache: *cache_save_brew_android
       - run:
           name: Install Android emulator
           shell: /bin/bash -e
           command: |
-            yes | sdkmanager "platform-tools" "tools" 1> /dev/null
-            yes | sdkmanager "platforms;android-28" "system-images;android-21;google_apis;x86" 1> /dev/null
-            yes | sdkmanager "emulator" --channel=3 1> /dev/null
-            yes | sdkmanager "build-tools;28.0.3" 1> /dev/null
-            yes | sdkmanager --licenses 1> /dev/null
+            yes | sdkmanager "platform-tools" "tools" >/dev/null
+            yes | sdkmanager "platforms;android-28" "system-images;android-28;default;x86_64" >/dev/null
+            yes | sdkmanager "emulator" --channel=3 >/dev/null
+            yes | sdkmanager "build-tools;28.0.3" >/dev/null
+            yes | sdkmanager --licenses >/dev/null
```

We shouldn't redirect `stderr` in case we need to handle them:
```suggestion
            yes | sdkmanager "platform-tools" "tools" 1> /dev/null
            yes | sdkmanager "platforms;android-28" "system-images;android-28;default;x86_64" 1> /dev/null
            yes | sdkmanager "emulator" --channel=3 1> /dev/null
            yes | sdkmanager "build-tools;28.0.3" 1> /dev/null
            yes | sdkmanager --licenses 1> /dev/null
```